modules/33acrossBidAdapter.js

```diff
@@ -6,7 +6,9 @@ const BIDDER_CODE = '33across';
 const END_POINT = 'https://ssc.33across.com/api/v1/hb';
 const SYNC_ENDPOINT = 'https://ssc-cms.33across.com/ps/?m=xch&rt=html&ru=deb';
 
-const adapterState = {};
+const adapterState = {
```

Comment:
We need to really re-visit this. Saving the adapter state i.e. auction state forever is bad practice. It has led to unit tests being stateful. 

For context, we had introduced `adapterState` in order to save the siteID of ad units passed through Bid Request phase in order to obtain them during user sync phase. The latter does not provide access (directly) to ad units.